autoware_adapi_v1_msgs/system/msg/MRMState.msg

```diff
@@ -0,0 +1,15 @@
+builtin_interfaces/Time stamp
+
+uint16 UNKNOWN = 0
+
+uint16 NORMAL = 1
+uint16 MRM_OPERATING = 2
+uint16 MRM_SUCCEEDED = 3
+uint16 MRM_FAILED = 4
+
+uint16 NONE = 101
+uint16 EMERGENCY_STOP = 102
+uint16 COMFORTABLE_STOP = 103
```

Comment:
I think it's better to add comments like Takagi-san did with other messages.
@isamu-takagi What do you think?

```suggestion
# For state
uint16 UNKNOWN = 0
uint16 NORMAL = 1
uint16 MRM_OPERATING = 2
uint16 MRM_SUCCEEDED = 3
uint16 MRM_FAILED = 4

# For behavior 
uint16 NONE = 101
uint16 EMERGENCY_STOP = 102
uint16 COMFORTABLE_STOP = 103
```